packages/koenig-lexical/src/plugins/AtLinkPlugin.jsx

```diff
@@ -147,6 +147,13 @@ export const KoenigAtLinkPlugin = ({searchLinks}) => {
 
                         // ensure we still have a cursor and it's inside the search node
                         atLinkNode.select(1, 1);
```

Comment:
@ronaldlangeveld were you able to change the offset here to fix the selection?

Reply:
Hi Kevin. That was one the first things I tried, but wasn't successful with it.